pkg/3scale/amp/component/system_options_builder.go

```diff
@@ -167,6 +179,10 @@ func (s *SystemOptionsBuilder) setNonRequiredOptions() {
 	defaultMemcachedServers := "system-memcache:11211"
 	defaultEventHooksURL := "http://system-master:3000/master/events/import"
 	defaultRedisURL := "redis://system-redis:6379/1"
+	defaultMessageBusRedisURL := "redis://system-redis:6379/8"
```

Comment:
```suggestion
	defaultMessageBusRedisURL := "redis://system-redis:6379/2"
```

so it matches the default behavior when `MESSAGE_BUS_REDIS_URL` is not defined.

Reply:
It can be empty by default as well.

Reply:
Updated to make the parameter not mandatory. The envvar in the system pods thus will be set, and will be set as the empty string